hs_labels/models.py

```diff
@@ -0,0 +1,848 @@
+__author__ = 'Alva'
+
```

Comment:
Can remove the __author__ and blank line from the start. Docstring should be the first thing and git tracks authorship.